docker-compose.yml

```diff
@@ -98,84 +97,109 @@ services:
     volumes:
       - kafdata:/var/lib/kafka
 
-  #---------------------------------------------- Services
+  # Services
+  # Landing with info
   portal:
     image: nginx:1.13.9-alpine
     volumes:
       - ./etc/landing:/usr/share/nginx/html
     ports:
       - 6300:80
 
-  # Airlock Gate - HTTP -> Kafka proxy
+  # Configurate main components (cassandra, zookeeper, kafka)
+  init:
+    image: vstk/hercules-init:0.38.0-SNAPSHOT
+    container_name: init
+    depends_on:
+      cassandra:
+        condition: service_healthy
+    volumes:
+      - ./etc/properties/init:/etc/hercules
+  
+  # Kafka producer    
   gate:
-    image: vstk/airlock.gate:latest
+    image: vstk/hercules-gate:0.38.0-SNAPSHOT
+    container_name: gate
+    depends_on:
+      - init
```

Comment:
Depends on `init` started, but not finished, let alone finished successfully. Is this a problem?

Reply:
Yes it can be a problem.